values.yaml

```diff
@@ -140,6 +135,11 @@ Lotus:
     is_slave: false
     label: some-release
     gateway: false
+    annotations:
+          prometheus.io/scrape: "true"
+          prometheus.io/port: "1234"
+          prometheus.io/path: /debug/metrics
+
```

Comment:
why did you add this annotation as default?
and why does it have so many spaces after annotations: ?

Reply:
I've added this annotation as default because it was originally part of service template file, thus I've decided that it will better to give us possibility to setup annotations in values file.

Reply:
if we have annotation which should be added everywhere.
make them to a default annotations block

if we have custom/specific annotations add them additionally to default.


we can have 2 variables one for default when we know that the default value spreads across all templates.
and the second one is custom. In case, when we want to add an extra annotation for one or a few services via overrides value files (live for space06)